docs/website/docs/dlt-ecosystem/destinations/filesystem.md

```diff
@@ -1,4 +1,4 @@
-# Cloud storage and filesystem
+# Cloud storage and file system
```

Comment:
Hm, not sure we need to change this, especially considering we call a source `filesystem`, not `file_system`

<img width="640" alt="image" src="https://github.com/user-attachments/assets/dba32d5b-5748-4700-aaf2-e4cefae91f3a">

Reply:
ok, rolled the change back